backend/src/models/user.js

```diff
@@ -40,6 +47,9 @@ class User {
      *                       contain the user data (probs eventually from an auth token)
      */
     createUser({ displayName, email, uid }) {
```

Comment:
Isn't this called after we hit the firebase api? This isn't as trivial as changing this field, because the email, uid will be potentially created by firebase even if our backend rejects. I'm going to look at this today and tomorrow in any case.

Reply:
I think this is fine... the email and uid may be valid, but this will stop them creating a _profile_ without a displayName